optuna/visualization/_pareto_front.py

```diff
@@ -121,6 +121,47 @@ def objective(trial):
 
     Returns:
         A :class:`plotly.graph_objs.Figure` object.
+
+
+    Example:
+
+        The following code snippet shows how to plot a 2-dimensional Pareto front
+        of a 3-dimensional study.
+        This example is scalable, e.g., for plotting a 2- or 3-dimensional Pareto front
+        of a 4-dimensional study and so on.
+        The example also contains the option to crop the displayed scale.
+
+        .. plotly::
+
+            import optuna
+
+            def objective(trial):
+                value_a = trial.suggest_int("a", 0,100)
+                value_b = trial.suggest_int("b", 0, 100)
+
+                v0 = value_a ** 2
+                v1 = value_b **2
+                v2 = value_a + value_b
+
+                return v0, v1, v2
+
+            study = optuna.create_study(directions=["minimize", "minimize", "minimize"],
+                            sampler=optuna.samplers.NSGAIISampler())
```

Comment:
You don't have to specify the `NSGAIISampler` since it is a default sampler for multi-objective optimization.

Reply:
resolved in bba7188527cf2e21b5992115956c9f22834a8860